tests/php/modules/shortcodes/test_class.slideshow.php

```diff
@@ -24,4 +24,43 @@ public function test_shortcodes_slideshow() {
 		$this->assertNotEquals( $content, $shortcode_content );
 	}
 
+	public function test_shortcodes_slideshow_no_js() {
+		switch_to_blog( 104104364 ); // test.wordpress.com
+
+		$content = '[gallery type="slideshow" ids="161,162"]';
+
+		$shortcode_content = do_shortcode( $content );
+
+		$this->assertEquals( 0, strpos( $shortcode_content, '<p class="jetpack-slideshow-noscript robots-nocontent">This slideshow requires JavaScript.</p>' ) );
```

Comment:
Probably should use `assertStringStartsWith()`

https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertStringStartsWith